modules/geo-layers/src/tile-layer/tile-2d-header.js

```diff
@@ -32,13 +36,27 @@ export default class Tile2DHeader {
     return result;
   }
 
+  async _loadData(getTileData) {
+    const {x, y, z, bbox, layerId} = this;
+
+    // Construct unique identifier for request scheduler handle
+    const id = `${layerId}-${x}-${y}-${z}`;
+    const requestToken = await requestScheduler.scheduleRequest(id, () => this.isVisible === true);
+
+    let result;
```

Comment:
Set to `null` to be explicit about return values and avoid returning `undefined`.